src/cctp-sdk/index.ts

```diff
@@ -32,112 +32,129 @@ interface AttestationStatus {
   attestation: string;
 }
 
+interface Authorization {
+  validAfter: number;
+  validBefore: number;
+  nonce: Uint8Array;
+  v: number;
+  r: string;
+  s: string;
+}
+
```

Comment:
Put interfaces and enums etc. into a separate folder called `types` and you can have `types/interfaces` and `types/enums`